clusters/hive/pools/cvp/cvp-ocp-4-13-next-amd64-aws-eu-west-3_clusterpool.yaml

```diff
@@ -0,0 +1,40 @@
+apiVersion: hive.openshift.io/v1
+kind: ClusterPool
+metadata:
+  creationTimestamp: null
+  labels:
+    architecture: amd64
+    cloud: aws
+    owner: cvp
+    product: ocp
+    region: eu-west-3
+    version: "4.13"
+    version_lower: 4.13.0-0
+    version_stream: 4-dev-preview
+    version_upper: 4.14.0-0
```

Comment:
I had some debate about whether we'd want an imageset+pool for each minor, or just a "4.next" one that always uses the newest 4.next payload (i.e. would automatically transitition from a 4.13.ec to a 4.14.ec once they become available).

I can go either way on it, but specifying the "version" on line 11 above becomes tricky if the version can evolve (unless we can just set it to "4.next", i wasn't sure what was a valid value for that label).

If we did want to go that way, the easiest thing to do would be to just set the version_upper to a really high value like `4.999.0-0` and then i think everything else should just work.